### PR TITLE
[Test] separate nnfw testcase

### PR DESCRIPTION
**Pull request by jaeyun-jung**
separate testcases for nnfw plugin (functions for filter-plugin and ml-api)

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_api.cc [hunk 1]
```diff
@@ -0,0 +1,957 @@
+/**
+ * @file        unittest_tizen_nnfw_runtime_api.cc
+ * @date        07 Oct 2019
+ * @brief       Unit test for Tizen nnfw tensor filter plugin.
```
Comment (dongju-chae) [inline tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_api.cc:4]: It's minor comment. I think we now need more specified `brief` comments in both `unittest_tizen_nnfw_runtime_api.cc` and `unittest_tizen_nnfw_runtime_raw.cc`

File: tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_api.cc [hunk 2]
```diff
@@ -0,0 +1,957 @@
...
+    while (!done && waiting_time < TEST_TIMEOUT_LIMIT) {  \
+      g_mutex_lock (&g_test_mutex);                       \
+      done = (*sink_called_cnt >= expected_cnt);          \
+      waiting_time += TEST_DEFAULT_SLEEP_TIME;            \
+      g_mutex_unlock (&g_test_mutex);                     \
+      if (!done)                                          \
+        g_usleep (TEST_DEFAULT_SLEEP_TIME);               \
+    }                                                     \
+    ASSERT_TRUE (done);                                   \
+  } while (0);
```
Comment (myungjoo) [inline tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_api.cc:32]: Remove the last semicolon.

File: tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_api.cc (hunk 1 above)
Comment (jaeyun-jung) [reply tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_api.cc]: updated

File: tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_api.cc (hunk 2 above)
Comment (jaeyun-jung) [reply tests/tizen_nnfw_runtime/unittest_tizen_nnfw_runtime_api.cc]: updated

**Review by helloahn — APPROVED** [review]
it looks more reasonable

**Review by myungjoo — APPROVED** [review]
LGTM

**Pull request merged**